src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -1203,6 +1221,18 @@ namespace Microsoft.Management.Deployment
             /// Authentication arguments used when downloading the package installer if authentication is required.
             AuthenticationArguments AuthenticationArguments;
         }
+
+        [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 13)]

```

Comment:
nit: the contract version comment in line 5 is still 1.12

Reply:
Correctly so; that comment means that it has already been incremented for 1.12.